.vscode/settings.json

```
@@ -0,0 +1,7 @@
+{
+  "search.exclude": {
```

want to add these too?

```suggestion
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.codeActionsOnSave": {
    "source.fixAll": "explicit"
  },
  "editor.formatOnSave": true,
  "search.exclude": {
```

---

i personally don't like those first two but i'll configure prettier yeah